data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
+  Each small molecule was run alone in solution and docked to Mpro to compute absolute free energies of binding.
+  Simulations were run using GROMACS-5.0.4 or GROMACS-2020 and are stored as compressed binary XTC files.
+  Absolute free energies were computed using an expanded ensemble scheme, using 40 lambdas to decouple the ligand interactions.
+  Free energy data for each alchemical lambda is stored in the md.log file, and found more concisely in pre-scraped pkl dataframes.
+  The dataset comprises several projects, each having a RUN*/CLONE*/result* directory structure.
+  Each PROJ represents a different dataset of small molecules. Each RUN represents a different small molecule,
+  each CLONE is a unique MD run differing in initial atomic velocities, and result is a fragment of the contiguous simulation.
+  For the case of ligand-only "L" simulations, each xtc represents 10ns of sampling, while each complex "RL" xtc trajectory is only 1ns.
+  More information about the simulation set-up can be found [here](https://github.com/FoldingAtHome/covid-moonshot/blob/master/folding-at-home_submissions/COVID%20Moonshot%20051220%20Release%20Announcement.pdf).
+  In order to find particular PROJ/RUN of interest, see the results and organization dataframes.
+  
+  *Entire dataset:* To get the raw trajectory files (XTCs) of the whole dataset (~10+ TB/project), you can use the [AWS CLI](https://aws.amazon.com/cli/):
```

Comment:
Can you lead with how to get distilled subsets, or a single trajectory for each ligand, or retrieve data for a single ligand? Do you have a master table of ligands somewhere?

Reply:
My master table of ligands is the organization.pkl, which shows identity, project, run information for all protease simulations. I am leading with organization, then topology file, where I show examples for downloading build directories for an entire project, build directory for a particular run, solvated structure file, and structure file with just solute. The raw datasets section similarly shows examples for grabbing varying amounts of trajectory data.